### ISSUE TITLE
[REC] Adrenaline Rush is being suggested even if already buffed

### ISSUE THREAD

**Issue by 182cm82kg22cm** [Hekili/hekili#3866]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

Rogue - Outlaw

### Describe the Issue

The script shows that adrenaline rush is only being recommended if it is inactive. (I think)
However, even if buffed, it is being recommended.

How can I make it being only recommended as intended? Regardless of the remaining seconds, only if inactive and available.

### How to Reproduce

Basically have it buffed, wait in stealth until available again.
Recommended to use, usable, though already buffed.

### Snapshot (Link)

https://pastebin.com/7NgXnKps

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/bBuzt4PQ1R7SFN5mAjL87H

**Comment by chevyn11** [comment]
adrenaline rush is used on cooldown as your most efficient off gcd builder (see improved adrenaline rush - gives max combo points).

**Comment by 182cm82kg22cm** [comment]
But it wouldn't make sense to recommend AR if it's already buffed for over 10 seconds, if it is going to run out eventually.
Might as well save it for the expected 30s downtime later.

**Comment by 182cm82kg22cm** [comment]
Anyway I'll just use it manually instead. Nevermind
